Misc fixes (#989)
arangodb/docs-hugo: site/content/arangodb/3.10/release-notes/version-3.0/incompatible-changes-in-3-0.md

  values, without the surrounding square brackets. Additionally, string
  array members were not enclosed in quotes in the result string:

  - `[ ]` was converted to `""`
  - `[ 1, 2, 3 ]` was converted to `"1,2,3"`
  - `[ "test", 1, 2 ]` was converted to `"test,1,2"`
  
- objects are now converted to their JSON-stringify equivalents, e.g.

  - `{ }` is converted to `"{}"`
  - `{ a: 1, b: 2 }` is converted to `"{\"a\":1,\"b\":2}"`
  - `{ test: "foobar" }` is converted to `"{\"test\":\"foobar\"}"`
    
  Previous versions of ArangoDB always converted objects into the string
  `"[object Object]"`.

This change also affects other parts in AQL that used `TO_STRING()` to implicitly
cast operands to strings. It also affects the AQL functions `CONCAT()` and 
`CONCAT_SEPARATOR()` which treated array values differently. Previous versions
of ArangoDB used the old `TO_STRING()` behavior and automatically flattened
array values in the first level of each passed array.

Now, there is no automatic flattening of arrays and arrays are cast using the
new `TO_STRING()` behavior. If you pass an array as the sole parameter, its

> members get concatenated - which seems the same as the previous flattening of

Reviewer: ```suggestion
members get concatenated - which is equivalent to the previous flattening of
```
